### PR TITLE
Use hostname for logging output

### PR DESCRIPTION
**Pull request by SpComb**
Kicks in after the `ValidateHost` phase

```
==> Reading instructions ...
==> Sharpening tools ...
==> Starting to craft cluster ...
==> Validate hosts @ 167.99.36.141 167.99.36.158 167.99.36.223
    [167.99.36.158] Checking sudo access ...
    [167.99.36.141] Checking sudo access ...
    [167.99.36.223] Checking sudo access ...
    [167.99.36.158] Gathering host facts ...
    [167.99.36.141] Gathering host facts ...
    [167.99.36.223] Gathering host facts ...
    [167.99.36.141] Validating distro and version ...
    [167.99.36.141] Validating host configuration ...
    [167.99.36.158] Validating distro and version ...
    [167.99.36.158] Validating host configuration ...
    [167.99.36.223] Validating distro and version ...
    [167.99.36.223] Validating host configuration ...
==> Migrate master @ terom-kupo-master
==> Configure hosts @ terom-kupo-master terom-kupo-node1 terom-kupo-node2
    [terom-kupo-master] Configuring essential packages ...
    [terom-kupo-node1] Configuring essential packages ...
    [terom-kupo-node2] Configuring essential packages ...
    [terom-kupo-node2] Configuring package repositories ...
    [terom-kupo-node1] Configuring package repositories ...
    [terom-kupo-master] Configuring package repositories ...
    [terom-kupo-master] Configuring netfilter ...
    [terom-kupo-master] Configuring container runtime (docker) packages ...
    [terom-kupo-node1] Configuring netfilter ...
    [terom-kupo-node2] Configuring netfilter ...
    [terom-kupo-node1] Configuring container runtime (docker) packages ...
    [terom-kupo-node2] Configuring container runtime (docker) packages ...
==> Configure cfssl @ terom-kupo-master
    [terom-kupo-master] Installing cfssl ...
==> Configure etcd certificate authority @ terom-kupo-master
    [terom-kupo-master] Configuring etcd certificate authority ...
    [terom-kupo-master] Caching certificate authority files to memory ...
==> Configure etcd @ terom-kupo-master
    [terom-kupo-master] Configuring etcd certs ...
    [terom-kupo-master] Configuring etcd ...
```

### CODE REVIEW

**Comment by jakolehm** [comment]
I like it.

**Comment by kke** [comment]
Might be confusing though if you create the machines through some sort of automation or cloning that sets the same hostname on all hosts.

**Comment by jakolehm** [comment]
> Might be confusing though if you create the machines through some sort of automation or cloning that sets the same hostname on all hosts.

Hostnames need to be unique, see: https://github.com/kontena/pharos-cluster/issues/307

File: lib/pharos/configuration/host.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Host < Dry::Struct
       attr_accessor :os_release, :cpu_arch, :hostname, :api_endpoint, :private_interface_address, :checks
 
       def to_s
-        address
+        hostname || address
```
Comment (jakolehm) [inline lib/pharos/configuration/host.rb]: This can be quite long in aws (fqdn). Should we just use a short hostname always?

Comment (SpComb) [reply lib/pharos/configuration/host.rb]: Done

**Comment by SpComb** [comment]
Merged to resolve conflicts. This is approved, do we include this in 1.3?

**Pull request merged**